src/news/release-1.4.0.md

```diff
@@ -0,0 +1,85 @@
+---
+title: PolyMC Update 1.4.0, now available
+description: Just Enough Changes
+date: 2022-07-17
+release_version: 1.4.0
+minimum_macos_version: 10.14.0
+mac_signature: epiclmao
+tags:
+  - release
+---
+
+Whoa! There have been a few things since our last feature release almost two months ago!
+
+Major new features include the new mod updater and Qt 6 support.
```

Comment:
```suggestion
Major new features include the new mod updater and Qt 6 support.
If you still want to use Qt 5 don't worry, in the short term PolyMC will still support it and packages for it are available.
However, if you use Qt 6 your experience should be better overall, especially for HiDPI users.
```

Reply:
I mean we only say "Qt 6 support", not "switch to Qt 6", so I don't think anyone would misunderstand it.

Reply:
I think some people will think we're switching to Qt 6 in any case, better to clarify that.
And if they don't think we're switching to qt6, some will think we want to remove qt5 support asap, when we aren't